charts/ratify/templates/akv-key-management-provider.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     helm.sh/hook-weight: "5"
 spec:
   type: azurekeyvault
+  {{- if .Values.azurekeyvault.refreshInterval }}
+  refreshInterval: {{ .Values.azurekeyvault.refreshInterval  }}
+  {{ end }}
```

Comment:
nit: `{{- end }}` is recommended to avoid redundant newline.
![image](https://github.com/user-attachments/assets/c8b528ce-d7cf-4d04-810b-3183d9cb0eff)